gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckUnusedDependenciesTask.java

```diff
@@ -117,9 +118,12 @@ public final void checkUnusedDependencies() {
 
                 if (!didYouMean.isEmpty()) {
                     sb.append("\t\tDid you mean:\n");
-                    didYouMean.forEach(transitive -> sb.append("\t\t\timplementation '")
-                            .append(asString(transitive))
-                            .append("\' \n"));
+                    didYouMean.stream()
+                            .map(BaselineExactDependencies::asString)
+                            .sorted()
+                            .forEach(transitive -> sb.append("\t\t\timplementation '")
+                                    .append(transitive)
+                                    .append("\' \n"));
```

Comment:
I think you can use `Collectors.joining` instead of manually building up a `StringBuilder` here.

Reply:
This is using the existing string builder for the entire output. I can collect to a string and then append to builder if you prefer since it's a bit cleaner, but just note that it is slightly less efficient.

Reply:
I'm fine with keeping this as is tbh